src/components/new-request/search.vue

```diff
@@ -240,7 +240,7 @@
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
         <BusinessLookupFetch />
-        <CompanyType v-if="getSearchBusiness && isBcRestorable && isSupportedRestoration(getEntityTypeCd)" />
+        <CompanyType v-if="getSearchBusiness && isBcRestorable && isNumberedEntityType" />
```

Comment:
I am wondering why the statement `isBcRestorable` is here... if the company is not restorable (due to its type) then CompanyType is not rendered, but what about the other components below?

1. Please test a non-restorable business type and see what components you see.

2. You may need to put some of these elements in a block with "v-if=isBcRestorable".

Reply:
This looks correct:

Reply:
This XP looks correct:

![image](https://github.com/bcgov/namerequest/assets/10002889/b6d9b7c1-a2b0-4f5e-9af1-a6a2ed2ca17d)

Reply:
I guess this code is OK. Resolving this conversation.